src/OpenTelemetry.Extensions.Enrichment.AspNetCore/OpenTelemetry.Extensions.Enrichment.AspNetCore.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetMinimumSupportedVersion)</TargetFrameworks>
+    <Description>OpenTelemetry .NET SDK ASP.NET Core telemetry enrichment.</Description>
+    <NoWarn>$(NoWarn),CS1591</NoWarn>
```

Comment:
Is this line really needed?
Can warnings/error be fixed?

Reply:
Fixed. Thank you